index.ts

```diff
@@ -65,6 +67,11 @@ async function main() {
                     // - pricing
                     // - repository name (devpool issue body contains a partner project issue URL)
                     // - bounty hunter assigned/unassigned an issue
+                    const issuePriceLabel  = getIssuePriceLabel(projectIssue) 
+                    if(issuePriceLabel === DEFAULT_PRICE_LABEL){
+                      console.log(`Pricing not set for the issue, skipping. ${projectIssue.html_url}`)
+                      return
```

Comment:
`return` breaks the [loop](https://github.com/ubiquity/devpool-directory/blob/f1b56a82543a911cc43295d59c4b2479755cce57/index.ts#L50), we should `continue` instead